save-orchestrator/src/main/kotlin/com/saveourtool/save/orchestrator/controller/AgentsController.kt

```diff
@@ -88,10 +96,11 @@ class AgentsController(
     }
 
     /**
+     * @param executionId ID of execution for which agents should be stopped
      * @param agentIds list of IDs of agents to stop
      */
     @PostMapping("/stopAgents")
-    fun stopAgents(@RequestBody agentIds: List<String>) {
+    fun stopAgents(@RequestParam executionId: Long, @RequestBody agentIds: List<String>) {
```

Comment:
why do we need executionId here?

Reply:
I wanted to move from stopping individual agents by ID to terminating the whole group of agents using executionId. Because, unlike previous setup with Docker, in Kubernetes we have an entity that acts as a grouping for agents - a Job. However, I've decided to postpone this change till later PRs. I'll remove this parameter for now